platform/lib/pipeline/componentReferenceImporter.js

```diff
@@ -198,8 +199,9 @@ class ComponentReferenceImporter {
     }
 
     return Promise.all(
-      documents.map((doc) => {
-        return this._createGrowDoc(doc);
+      documents.map(async (doc) => {
```

Comment:
any concern about this failing? Or would the function calling the promise be catching that?

Reply:
good catch, forgot to clean this up